lib/service/connect.go

```diff
@@ -117,6 +117,9 @@ func (process *TeleportProcess) reconnectToAuthService(role types.SystemRole) (*
 			process.log.Error("Can not join the cluster as node, the token expired or not found. Regenerate the token and try again.")
 		case connectErr != nil:
 			process.log.Errorf("%v failed to establish connection to cluster: %v.", role, connectErr)
+			if process.Config.Version == defaults.TeleportConfigVersionV3 {
```

Comment:
Do we do a better check here? We could check the error message for `transport: authentication handshake failed:`, but I'm not sure if it's better to produce this tip more often than not

Reply:
I think we can leverage `process.Config.ProxyServer.IsEmpty()` to cut down on false positives.